WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -512,6 +522,16 @@ public async Task<IActionResult> PostOutputAsync([FromQuery, Required] long roun
 				}
 			}
 
+			if (round.GetAlicesBy(request.OutputAddress).ToArray().Length != 0)
```

Comment:
concept NACK, a legitimate usecase: if someone is mixing from donations and someone else is donating in the same mix. Note this'd also make the round fail.

Reply:
While I agree there is a legitimate usecase it is far more likely to be used malicously